geomesa-hbase/geomesa-hbase-dist/pom.xml

```diff
@@ -93,6 +93,11 @@
             <artifactId>slf4j-log4j12</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <scope>compile</scope>
```

Comment:
actually having 2 log4j jars might not matter - it's only having 2 slf4j jars that messes things up. it would probably be good to exclude log4j from findjars, but we'd have to make sure all our distros bundle it